### PR TITLE
bitfetch: new package at 2.3

### PR DESCRIPTION
**Pull request by bit9tream**
## Description of package

Simple cli system information tool written in C.

## Installed manifest of package

```
/var/db/kiss/installed/bitfetch/version
/var/db/kiss/installed/bitfetch/sources
/var/db/kiss/installed/bitfetch/manifest
/var/db/kiss/installed/bitfetch/depends
/var/db/kiss/installed/bitfetch/checksums
/var/db/kiss/installed/bitfetch/build
/var/db/kiss/installed/bitfetch/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/bitfetch
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/bitfetch/build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make ID=kiss bitfetch-build
+make DESTDIR="$1" PREFIX="/usr" install
```
Comment (dylanaraps) [inline community/bitfetch/build:4]: Quotes unneeded for `/usr`.

Comment (bit9tream) [reply community/bitfetch/build:4]: I know, but I'm just used to writing like that.

Comment (dylanaraps) [reply community/bitfetch/build:4]: Sure. It's a requirement that all build files in the "official" repositories follow the same style (which is why I request changes like this).

Comment (bit9tream) [reply community/bitfetch/build:4]: Oh, ok.
I'll change that.

**Pull request merged**